sympy/calculus/codomain.py

```diff
@@ -151,3 +154,73 @@ def codomain_interval(f, set_val, *sym):
     if isinstance(domain, FiniteSet):
         return FiniteSet(*[limit(func, symbol, i) if i in FiniteSet(-oo, oo)
                             else func.subs({symbol: i}) for i in domain])
+
+
+def not_empty_in(fin_set, intrvl, *syms):
+    """ Finds the domain in which the Set input is not empty
```

Comment:
`fin_set` --> `finite_set` and `intervl` --> `interval` or `sets` (Explicit is better than implicit)

The first argument of the following can only be `FiniteSet`, I guess, If so then you mean the following:

```
Finds the domain of the function(s) in `finite_set` in which the `finite_set` is not empty,
given it's range as `interval`.
```
or
```
Finds the domain of the function(s) in `finite_set` in which the `finite_set` is a subset of the `interval`.
```